sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -1940,6 +1943,7 @@ public partial class VirtualMachineScaleSetData : Azure.ResourceManager.Models.T
         public VirtualMachineScaleSetData(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
         public Azure.ResourceManager.Compute.Models.AdditionalCapabilities AdditionalCapabilities { get { throw null; } set { } }
         public Azure.ResourceManager.Compute.Models.AutomaticRepairsPolicy AutomaticRepairsPolicy { get { throw null; } set { } }
+        public bool? ConstrainedMaximumCapacity { get { throw null; } set { } }
```

Comment:
Could we rename this property to something like `IsMaximumCapacityConstrained`?
There is a general arch board comment that boolean value property names should start with `Is` when possible